test/packages/bad_github_owner/changelog.yml

```diff
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.0.1"
+  changes:
+    - description: Initial draft of the package
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/0 # FIXME Replace with the real PR link
```

Comment:
Replace this with https://github.com/elastic/package-spec/pull/164?

Reply:
Not necessary, this package has been created with `elastic-package create package`. We can leave it here ;)

Reply:
I know, just as a convention I've been using PRs where a test package was introduced as the link in that test package's changelog. You can see it in some other test packages. That way its easy to understand _why_ a test package was introduced (what was the context for it).

Reply:
But this is not a hard requirement, just a convention. So if you want to leave this as-is, that's fine with me.

Reply:
Makes sense, let me update it quickly!

Reply:
Fixed